src/Header/Nav/SubNav/SubNav.module.scss

```diff
@@ -118,4 +118,5 @@
 
 .image {
   height: 100%;
+  padding: utils.rem(spacing.$small + spacing.$x-small);
```

Comment:
Just checking if we only wanted to apply top and bottom padding?  Currently applies horizontal padding too.

Reply:
I swear I remember typing the 0 for the L/R value🤦  Added now!